content/en/cosign/overview.md

```diff
@@ -63,24 +63,74 @@ $ cosign verify-blob <file> --bundle cosign.bundle --certificate-identity=name@e
+
+To use `ttl.sh` and [crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane) to prepare the image that we want to sign. Run the following:
+
+```
+$ SRC_IMAGE=busybox
+$ SRC_DIGEST=$(crane digest busybox)
+$ IMAGE_URI=ttl.sh/$(uuidgen | head -c 8 | tr 'A-Z' 'a-z')
+$ crane cp $SRC_IMAGE@$SRC_DIGEST $IMAGE_URI:1h
+$ IMAGE_URI_DIGEST=$IMAGE_URI@$SRC_DIGEST
+```
+
+### Keyless signing of a container
+
+The following code signs the created container image.  The command to sign container images is `cosign sign <image URI>`.  Note that for containers, there is no bundle as there is with blobs, as the signature and certificate are attached directly to the container:
 
 ```
-$ cosign sign <image URI>
+$ cosign sign $IMAGE_URI_DIGEST
+Generating ephemeral keys...
+Retrieving signed certificate...
+Successfully verified SCT...
+
+	Note that there may be personally identifiable information associated with this signed artifact.
+	This may include the email address associated with the account with which you authenticate.
+	This information will be used for signing this artifact and will be stored in public transparency logs and cannot be removed later.
+
+By typing 'y', you attest that you grant (or have permission to grant) and agree to have this information stored permanently in transparency logs.
+Are you sure you would like to continue? [y/N] y
+tlog entry created with index: 18914440
+Pushing signature to: ttl.sh/4d6d55ae
 ```
 
-This works the same as signing a blob, but the signature and certificate are attached as container metadata.
+### Keyless verifying of a container
 
-To verify a signed container image, use the following command:
+This works the similar to verifying a blob, but there is no need to place the certificate and signature on the verify command. To verify a signed container image, use the following command:
 
 ```
 $ cosign verify <image URI> --certificate-identity=name@example.com
                             --certificate-oidc-issuer=https://accounts.example.com
 ```
 
-### Signing with a generated key
+> Note that for our example we use the `regexp` versions of the identity options:
+
+```
+cosign verify $IMAGE_URI_DIGEST --certificate-identity-regexp=.* --certificate-oidc-issuer-regexp=.*
+
+Verification for ttl.sh/4d6d55ae@sha256:b5d6fe0712636ceb7430189de28819e195e8966372edfc2d9409d79402a0dc16 --
+The following checks were performed on each of these signatures:
+  - The cosign claims were validated
+  - Existence of the claims in the transparency log was verified offline
+  - The code-signing certificate was verified using trusted certificate authority certificates
+
+[{"critical":{"identity":{"docker-reference":"ttl.sh/4d6d55ae"},"image":{"docker-manifest-digest":"sha256:b5d6f2a0dc1"
+
+...
+```
+
+The rest of the flags (annotations, claims, tlog, etc.) all work the same.
```

Comment:
Can we delete lines 125-132, these three paragraphs? All of this information is already discussed in the intro to the quickstart.

Reply:
removed